src/Type/Enum/ImageSizeEnum.php

```diff
@@ -0,0 +1,7 @@
+<?php
+namespace WPGraphQL\Type;
+
+register_graphql_enum_type( 'ImageSizeEnum', [
+  'description' => __( 'Size of image', 'wp-graphql' ),
+  'values' => array( 'THUMBNAIL', 'MEDIUM', 'LARGE', 'FULLSIZE' )
```

Comment:
square bracket arrays, and I _think_ we probably want these to have values? like: 

```
'values' => [
	'THUMBNAIL' => [
		'value' => 'thumbnail',
	], 
	'MEDIUM' => [
		'value' => 'medium',
	], 
	'LARGE' => [
		'value' => 'large',
	],
	'FULLSIZE' => [
		'value' => 'fullsize',
	],
]
```

Reply:
actually, come to think of it, this Enum should probably be generated from the sizes that have been registered to the WP Install, eh? via something like: https://codex.wordpress.org/Function_Reference/get_intermediate_image_sizes